schema/config-schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "description": "OpenContainer Config Specification",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://opencontainers.org/schema/image/config",
+  "type": "object",
+  "properties": {
+    "created": {
+      "type": "string",
+      "format": "date-time"
```

Comment:
On Mon, May 23, 2016 at 01:51:33PM -0700, Vincent Batts wrote:
> > +    "created": {
> > +      "type": "string",
> > +      "format": "date-time"
> 
> is this format too strict for `"2015-10-31T22:22:56.015925234Z"`?

The v4 JSON Schema RFC doesn't mention date-time [1], but it's listed
here [2] referencing RFC 3339, section 5.6.  And your string is
compatible with that [3].

[1]: https://tools.ietf.org/html/draft-zyp-json-schema-04
[2]: http://json-schema.org/latest/json-schema-validation.html#anchor108
[3]: http://tools.ietf.org/html/rfc3339#section-5.6